src/controllers/authenticationController.ts

```diff
@@ -77,7 +79,7 @@ export default class AuthenticationController {
    */
   public async logout(
     req: express.Request
-  ): Promise<IResponseErrorInternal | IResponsePermanentRedirect> {
+  ): Promise<IResponseErrorInternal | IResponseSuccessJson<{}>> {
```

Comment:
I would, at least, respond with a simple

```json
{
  "result": "OK"
}
```

Reply:
The notification service uses:

```
value: { message: "ok" }
```

can we use the same format?

Reply:
yes sure